nixos/modules/services/monitoring/prometheus/exporters/artifactory.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.artifactory;
+in
+{
+  port = 9531;
```

Comment:
Port already defined in the table of default allocations:

https://github.com/prometheus/prometheus/wiki/Default-port-allocations